README.md

```diff
@@ -78,9 +78,22 @@ some changes you can just run `make reload-data` without having to stop inframoc
 
 ## Migrating SQL data
 
-Knex is the package used to apply SQL migrations. The package configuration is done in the API. If you want to apply the latest migrations in your local `api` repo to the your local SQL instance you can run either do:
-1. `make migrate`: this will use the API configuration but you need to have knex command available. You can install it `npm install knex`.
-2. Go to `api/src/sql` and run `knex migrate:latest`.
+Knex is the Node.js package used to apply SQL migrations. The `knex` command configuration is done in the `api` repo. There are `make` commands available in `inframock` which references these `knex` commands. However, it assumes that the `api` folder is in the same level (i.e. contained in the same folder) as the `inframock` folder, so make sure that this is the case for your installation before continuing.
```

Comment:
about this:
```
However, it assumes that the `api` folder is in the same level (i.e. contained in the same folder) as the `inframock` folder, so make sure that this is the case for your installation before continuing.
```
I checked the makefile, and it references iac. Examples:
```
migrate:
	@NODE_ENV=development AWS_ACCOUNT_ID=000000000000 AWS_REGION=eu-west-1 knex migrate:latest --cwd ../iac/migrations/sql-migrations/
migrate-down:
	@NODE_ENV=development AWS_ACCOUNT_ID=000000000000 AWS_REGION=eu-west-1 knex migrate:rollback --all --cwd ../iac/migrations/sql-migrations/
```
So the documentation here should also have the same note that the iac folder has to be under the same folder as inframock and the api. Or am I missing something?

Reply:
oh right, we're not using the api repo anyore. Migration files are copied from the API repo to IAC (see https://github.com/biomage-org/api/blob/b4dc5e59f8cad24c9ca75d4d6e9b7389b8e63995/.github/workflows/ci.yaml#L364). We then use the migrations in IAC. Great catch Iva! I'll update this README